pyccel/commands/pyccel_make.py

```diff
@@ -0,0 +1,163 @@
+    compiler_group.add_argument('--compiler-config',
+                                type=Path,
+                                default=None,
+                                metavar='CONFIG.json',
+                                help='Load all compiler information from a JSON file with the given path (relative or absolute).')
+    group.add_argument('--debug', action=argparse.BooleanOptionalAction, default=None,
+                        help='Compile the code with debug flags, or not.\n' \
+                        ' Overrides the environment variable PYCCEL_DEBUG_MODE, if it exists. Otherwise default is False.')
+    group.add_argument('--output', type=Path, default = None,\
+                       help="Folder in which the output is stored (default: FILE's folder).")
+
+    # ... Accelerators
+    group = parser.add_argument_group('Accelerators options')
+    group.add_argument('--mpi', action='store_true', \
+                       help='Use MPI.')
+    group.add_argument('--openmp', action='store_true', \
+                       help='Use OpenMP.')
+#    group.add_argument('--openacc', action='store_true', \
+#                       help='Use OpenACC.') # [YG 17.06.2025] OpenACC is not supported yet
+    # ...
+
+    # ... Other options
+    group = parser.add_argument_group('Other options')
+    group.add_argument('-v', '--verbose', action='count', default = 0,\
+                        help='Increase output verbosity (use -v, -vv, -vvv for more detailed output).')
+    group.add_argument('--time-execution', action='store_true', \
+                        help='Print the time spent in each section of the execution.')
+    group.add_argument('--developer-mode', action='store_true', \
+                        help='Show internal messages.')
+    group.add_argument('--conda-warnings', choices=('off', 'basic', 'verbose'), default='basic',
+                        help='Specify the level of Conda warnings to display (default: basic).')
+    group.add_argument('-t', '--convert-only', action='store_true',
+                       help='Stop Pyccel after translation to the target language, before build.')
+    # ...
+    # ...
+    args = parser.parse_args()
+
+    from pyccel.codegen.make_pipeline  import execute_pyccel_make
+    from pyccel.errors.errors     import Errors, PyccelError
+    from pyccel.errors.errors     import ErrorsMode
+
+    errors = Errors()
+
+    if args.files:
+        files = args.files
+    elif args.glob:
+        files = [Path(f) for f in glob.glob(args.glob, recursive=True)]
+    elif args.file_descr:
+        with open(args.file_descr, 'r', encoding='utf-8') as f:
+            files = [Path(fname.strip()) for fname in f.readlines()]
+
+        for f in files:
+            if not f.exists():
+                errors.report(f"File not found : {f}", severity='error')
+
+        cwd = os.getcwd()
+        files = [f.relative_to(cwd) if f.is_absolute() else f for f in files]
```

Comment:
Should we not execute this code also in the case that `--files` is given? Further, I would check that the paths are those of actual `*.py` files, not folders or anything else.